Tests/TrustylibTests/TestResources/authentication_service_response.json

```diff
@@ -0,0 +1,8 @@
+{
+    "access_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJpd0hzYW1feTRYUUlBVzQ1bHEtR2xMWWRPN1BfNUp2ZVdSeC0tM1kyeFlVIn0.eyJleHAiOjE2NzYzOTIyNTIsImlhdCI6MTY3NjM4ODY1MiwianRpIjoiYzU3YWU3ZTQtM2EyNy00Njc0LWIwOWItY2MyZjBlNzQxYzVmIiwiaXNzIjoiaHR0cHM6Ly9sb2dpbi5ldHJ1c3RlZC5jb20vYXV0aC9yZWFsbXMvYnVzaW5lc3MiLCJhdWQiOiJiMmItY3VzdG9tLWNsaWVudCIsInN1YiI6IjRjY2RmOGI1LTljYzQtNDhjYy04NGNiLTg3OWI1MzhiN2E4ZiIsInR5cCI6IkJlYXJlciIsImF6cCI6ImQxOTdlZDQxZGI4NF9fZXRydXN0ZWQtaW9zLXNkayIsInNjb3BlIjoidHNfY2xpZW50cyIsImFyZWFzLWFsbG93ZWQiOlsibWFycyJdLCJjbGllbnRJZCI6ImQxOTdlZDQxZGI4NF9fZXRydXN0ZWQtaW9zLXNkayIsImNsaWVudEhvc3QiOiI1NC44Ni41MC4xMzkiLCJodHRwczovL2V0cnVzdGVkLmNvbSI6eyJhY2NvdW50UmVmIjoiYWNjLWQxOTdlZDQxLWRiODQtNGVkZi04OTgxLWZjMGUyMTdhNGJhYiIsInJvbGUiOiJvd25lciIsInBlcm1pc3Npb25zIjp7InVybiI6InVybjpldHJ1c3RlZDoqIn19LCJpZGVudGl0eVJlZiI6Imlkbi00OWQ5ZWY0NC0xN2EyLTQ3ZTAtYTU5NC01NWMzNzY1ZTIzMjQiLCJjbGllbnRBZGRyZXNzIjoiNTQuODYuNTAuMTM5In0.VsOl8_OH6WONBwk4o2_XjKdv01zTfLdKrBur1uO2CJ7gDON9SrFHF96snA4_vmsRKDbz01y61V13c-644HlOBVrTnZiXBw0s1U9mAjaONR8jeqvWm1IrD6dIQLTx28Ln04B3QhYCcoG619cAVLI5uFFOLkR4fpHfDOysoDetKsdtmaKIZG0teoHh9nLADEG3rDmfzzAOOQNnGAcwt271QDppDRIA0LXF6bnM2P9_KrM98yXPo0wN3AyczvqoHqlpQSe9TyCG_qfcS0KIQSds6Fyar4aTyJLAcpVEN47pmEEap34oOlcUrJ8-TQOzmb_2CmbmqcS6wz_5k9YJ2un_yA",
+    "expires_in": 3600,
+    "refresh_expires_in": 0,
+    "token_type": "Bearer",
+    "not-before-policy": 1651869373,
+    "scope": "ts_clients"
+}
```

Comment:
question: is this a real token with validity?

Reply:
Nice catch @superus8r :)
Though the API response is old and already expired but its wise to change it to prevent any security related lapse. Thank you.

Reply:
Just a suggestion: You can also create a "fake" token using [jwt.io](https://jwt.io/), so just pasting the token in there, change the signature, and you are on the safe side of things :)

Reply:
Thank you @timo-reymann for this useful insight 👍

Reply:
hey @superus8r , @timo-reymann as suggested, I replaced the auth token and expires in values with mock ones. Here are the changes - https://github.com/trustedshops-public/etrusted-ios-trustbadge-library/pull/6/commits/9c4ebf182aa98aaf9b7e39df0bedf8a66b21952a